### PR TITLE
B 01085/go to top

### PR DESCRIPTION
**Pull request by dphamsd**
When scrolling down a button will appear allowing to scroll back to top.

### CODE REVIEW

File: public/css/styles.css [hunk 1]
```diff
@@ -131,6 +131,19 @@ body {
...
+.to-top {
+  position: fixed;
+  bottom: 10px;
+  right: 5px;
+
+  padding: 1em;
+
+  display: none;
+}
+
```
Comment (jasonmullins) [inline public/css/styles.css:141]: remove extra line breaks

File: public/js/util/backtotop.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+       jQuery('.to-top').fadeOut(duration);
+     }
+  });
+
+  jQuery('.to-top').click(function(event) {
+     event.preventDefault();
+     jQuery('html, body').animate({scrollTop: 0}, duration);
+     return false;
+    })
+});
```
Comment (jasonmullins) [inline public/js/util/backtotop.js]: add newline (setup your editor to automatically add this)

File: views/index.jade [hunk 3]
```diff
@@ -1,5 +1,8 @@
 extends ./layouts/main.layout.jade
 block header
   include ./header/header.jade
+
 block main
   div(ng-view)
+  div.to-top
+    a.btn.btn-primary.glyphicon.glyphicon-arrow-up
```
Comment (jasonmullins) [inline views/index.jade]: Add newline

**Comment by dphamsd** [comment]
Oop, leftover comment, that's supposed to be fixed lines and set default

**Comment by krullshards** [comment]
:+1:

**Comment by jasonmullins** [comment]
:thumbsup:

**Pull request merged**